apps/build/src/job.ts

```diff
@@ -15,6 +16,13 @@ export const performBuild = async (build: Build) => {
       .map(({ id }) => screenshotDiffJob.push(id))
   );
 
+  const account = await Project.relatedQuery<Account>("account")
+    .for(build.projectId)
+    .first()
```

Comment:
no need

Reply:
Heu... without `.first()`, it return an array of accounts. So, I think it's useful